packages/ui/src/utils/ThemeUtil.ts

```diff
@@ -1,231 +1,433 @@
-import { css } from 'lit'
+import { css, unsafeCSS } from 'lit'
+import type { ThemeVariables } from './TypesUtil.js'
 
 // -- Utilities ---------------------------------------------------------------
-export function initializeTheming() {
+export function initializeTheming(themeVariables: ThemeVariables) {
   const styleTag = document.createElement('style')
+  styleTag.classList.add('w3m')
```

Comment:
why do we need to set class here in addition to dataset / querying by dataset latter on as well? Also, if we do need this class it should be `wui-something` not w3m.